build.sbt

```diff
@@ -73,6 +77,39 @@ lazy val codacyAnalysisModels = project
   .disablePlugins(com.lucidchart.sbt.scalafmt.ScalafmtCorePlugin)
   .enablePlugins(JavaAppPackaging)
 
+lazy val apiSwaggerFile: File =
+  codacyApiClient.base / ".."/ "api.yml"
```

Comment:
We can remove this now. We should be fetching this file from bintray or some other repository